test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -2078,6 +2084,339 @@ public void PushCommand_PushToServerV3_WithSymbols_ApiKey_SymbolApiKey_BothFromC
+        }
+
+        [Fact]
+        public void PushCommand_ConfigFile_NotFound()
+        {
+            var configFileName = "My.Config";
+
+            // Act
+            var args = new string[]
+            {
+                Command_Push,
+                "testPackage1",
+                Param_ConfigFile,
+                configFileName
+            };
+
+            var result = CommandRunner.Run(
+                            NuGetExePath,
+                            CurrentDirectoryPath,
+                            string.Join(" ", args),
+                            true);
+
+            // Assert
+            Assert.True(
+                result.ExitCode != 0,
+                "The run did not fail as desired. Simply got this output:" + result.Output);
+
+            var expectedError = string.Format(
+                                    CultureInfo.InvariantCulture,
+                                    "File '{0}' does not exist.",
+                                    Path.Combine(CurrentDirectoryPath, configFileName).ToString());
+
+            Assert.True(
+                result.Errors.Contains(expectedError),
+                string.Format(
+                    CultureInfo.InvariantCulture,
+                    "Expected error: {0} \r\nActual error: {1}",
+                    expectedError, result.Errors));
+        }
+
+        [Fact]
+        public void PushCommand_ConfigFile_BadFormat()
+        {
+            var configFileName = "My.Config";
+            var repositoryKey = "MySource";
+
+            using (var testDirectory = TestDirectory.Create())
+            {
+                var repositoryPath = Path.Combine(testDirectory, "repository");
+                var configFilePath = Path.Combine(testDirectory, "config");
+                Directory.CreateDirectory(repositoryPath);
+                Directory.CreateDirectory(configFilePath);
+
+                var packageFileName = Util.CreateTestPackage("testPackage1", "1.1.0", testDirectory);
+
+                FileSystemUtils.CreateFile(
+                    configFilePath,
+                    configFileName,
+                    string.Format(
+                        CultureInfo.InvariantCulture,
+                        @"Not XML!"));
+
+                var args = new string[]
+                {
+                    Command_Push,
+                    packageFileName,
+                    Param_Source,
+                    repositoryKey,
+                    Param_ConfigFile,
+                    Path.Combine(configFilePath, configFileName)
+                };
+
+                var result = CommandRunner.Run(
+                                NuGetExePath,
+                                testDirectory,
+                                string.Join(" ", args),
+                                true);
+
+                // TODO: The bad config file is ignored, so I need to confirm this is expected behavior.
```

Comment:
@zivkan , @nkolev92 - Any input here?

Reply:
As mentioned in chat, it does feel like a bug.  My first guess would have been that the config file isn't actually being read, but then I'd expect the next test, `PushCommand_ConfigFile_ExpectedFormat`, to fail.

Anyway, this PR clearly didn't introduce that behaviour, so we should create a new issue, and skip this test until that other issue is fixed.